### PR TITLE
post: release of js-libp2p 0.26

### PR DESCRIPTION
**Pull request by jacobheun**

### CODE REVIEW

File: content/post/055-js-libp2p-0.26.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+> Gossipsub support and Promisify API
+
+# 🔦 Highlights
+
+## 🗣 Gossipsub
+Thanks to the awesome work of the [ChainSafe](https://github.com/ChainSafe) team, [Gossipsub](https://github.com/ChainSafe/gossipsub-js) is here! If you are using Pubsub you can now switch to using Gossipsub instead of Floodsub. Have old peers you need to flood stuff to? Don't worry, Gossipsub will automatically fallback to Floodsub for peers that don't support it. See the API Changes section below for how to migrate your config over to using Gossipsub.
+
+## 🚉 Promisify
+As we migrate to async/await we are promisifying the Libp2p public methods. Several lower level libraries are currently leveraging libp2p for testing. Their transition to full async/await will be greatly helped by promisifying the libp2p api. Once the async/await changes are propagated up, we will remove Promisify in a future update, along with callback support, as libp2p will have full async/await support at that time. See the API Changes below to check out the Breaking Changes.
```
Comment (alanshaw) [inline content/post/055-js-libp2p-0.26.md]: Link to the async/await endeavour issue?

File: content/post/055-js-libp2p-0.26.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+```js
+libp2p.start(onStart)
+// or
+await libp2p.start()
+```
+
+## Gossipsub
+
+The Gossipsub API is integrated into the existing `libp2p.pubsub` api. Once Gossipsub has been supplied as your pubsub implementation, you will be able to use it just as Floodsub was previously used. See the section above for pubsub configuration changes.
```
Comment (alanshaw) [inline content/post/055-js-libp2p-0.26.md]: Link to the gossipsub API docs?

File: content/post/055-js-libp2p-0.26.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+# ❤️ Huge thank you to everyone that made this release possible
+
+In alphabetical order, here are all the humans that contributed to the release:
```
Comment (alanshaw) [inline content/post/055-js-libp2p-0.26.md]: FYI, I tweaked [the script](https://gist.github.com/alanshaw/5a2d9465c5a05b201d949551bdb1fcc3) to output total people and contribution counts like:

```js
`In alphabetical order, here are the ${peoples.length} humans that made ${totalContribs} contributions to this release:\n`
```

I also found a bug in rate limit detection for the `name-your-contributors` module so hopefully that'll be fixed soon https://github.com/mntnr/name-your-contributors/pull/108

File: content/post/055-js-libp2p-0.26.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    denyTTL: 120e3,
+    denyAttempts: 5,
+  }
+  ...
+})
+```
+
+#### Pubsub
+
+**Subscribe**: To comply with the pubsub interface, the order of params for `libp2p.pubsub.subscribe` have been reordered. They were previously `topic, options, handler, callback`, and are now `topic, handler, options, callback`. If your implementation does not use `options` you should be able to ignore this change.
```
Comment (alanshaw) [inline content/post/055-js-libp2p-0.26.md]: "pubsub interface"? Does this mean the interface in `interface-ipfs-core` or something else?

File: content/post/055-js-libp2p-0.26.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+title: js-libp2p 0.26.0 released
+author: Jacob Heun
+---
+
+> Gossipsub support and Promisify API
+
+# 🔦 Highlights
+
+## 🗣 Gossipsub
+Thanks to the awesome work of the [ChainSafe](https://github.com/ChainSafe) team, [Gossipsub](https://github.com/ChainSafe/gossipsub-js) is here! If you are using Pubsub you can now switch to using Gossipsub instead of Floodsub. Have old peers you need to flood stuff to? Don't worry, Gossipsub will automatically fallback to Floodsub for peers that don't support it. See the API Changes section below for how to migrate your config over to using Gossipsub.
```
Comment (alanshaw) [inline content/post/055-js-libp2p-0.26.md]: What is the difference, why is gossipsub better?

File: content/post/055-js-libp2p-0.26.md (hunk 1 above)
Comment (alanshaw) [inline content/post/055-js-libp2p-0.26.md]: Please can we consistently have "api" or "API" in this post?

File: content/post/055-js-libp2p-0.26.md (hunk 4 above)
Comment (jacobheun) [reply content/post/055-js-libp2p-0.26.md]: Yes, `interface-ipfs-core`, I will link to it.

File: content/post/055-js-libp2p-0.26.md (hunk 2 above)
Comment (jacobheun) [reply content/post/055-js-libp2p-0.26.md]: I reworded this. The intent is to convey that Gossipsub works with the existing Pubsub API, so I have linked to those docs.

File: content/post/055-js-libp2p-0.26.md (hunk 5 above)
Comment (jacobheun) [reply content/post/055-js-libp2p-0.26.md]: Added some brief text explaining the main difference and linked to the Spec that has a much more detailed overview.

**Pull request merged**